### ISSUE TITLE
Settings don't affect all user accounts on a system.

### ISSUE THREAD

**Issue by Yoshlix** [ChrisTitusTech/winutil#2434]
## Describe the bug
When turning features on or off (e.g. show file extensions) it will not affect all user accounts on a system. To be clear, only the admin user account appears to be affected by the settings you make. Also, in the "Install" sections, the script doesn't detect the installed packages for the currently logged in user.

## To Reproduce
Log in with a normal account and run the script with your admin account. Then try to get the installed packages or change preferences under "Tweaks".

## Expected behavior
The script should work for the logged in user, even if he is not admin

**Comment by YusufKhalifadev** [comment]
@Yoshlix Can you specify what windows version are you on and what system do you have ?

**Comment by Yoshlix** [comment]
Windows 11 Pro, 23H2 (22631.3880)
CPU: Intel 12700k
GPU: Nvidia RTX 3060
Ram: 32GB

@YusufKhalifadev Any other information needed?

**Comment by Yoshlix** [comment]
@YusufKhalifadev So I have two user accounts on my system. On account with normal user rights. Thats the account I usally use. Than I have a second account with admin rights. Logged in with my normal account I start an elevated powershell terminal to start the script.

**Comment by YusufKhalifadev** [comment]
@Yoshlix Okay it's solved then because the script run at a user level so the tweaks made change per-user not system-wide so when chris made the script it was intended to run with admin in the normal user not with a separate admin account that's how to fix it remove your admin account and start the script as admin with your normal user.

**Comment by Yoshlix** [comment]
Yeah, that makes sense, but I think it should be possible to change these things system wide. Anyway, thanks for your help!
